corehq/apps/userreports/sql/adapter.py

```diff
@@ -208,6 +214,12 @@ def bulk_delete(self, docs, use_shard_col=True):
         with self.session_context() as session:
             session.execute(delete)
 
+        register_data_source_row_change(
+            domain=self.config.domain,
+            data_source_id=self.config._id,
+            row_changes=[{"doc_id": doc['_id']} for doc in docs],
+        )
+
     def delete(self, doc, use_shard_col=True):
```

Comment:
~~It looks like single doc deletes don't use this:~~

~~https://github.com/dimagi/commcare-hq/blob/6b0021c4a3b753a8beeac2f0e748abf2c575b6b7/corehq/apps/userreports/pillow.py#L540~~

Scratch that, it's just bad variable naming.